src/pkg/packager/creator/normal.go

```diff
@@ -490,12 +536,19 @@ func (pc *PackageCreator) addComponent(ctx context.Context, component v1alpha1.Z
 			}
 		}
 		spinner.Success()
+		l.Debug("done building k8s manifests",
```

Comment:
Thoughts on changing to "done processing k8s manifests", we only build manifests if kustomizations are used.